src/amo/components/AutoSearchInput/index.js

```diff
@@ -264,14 +265,14 @@ export class AutoSearchInputBase extends React.Component<InternalProps, State> {
 
   renderSuggestion = (suggestion: SuggestionType) => {
     const { loadingSuggestions, selectSuggestionText } = this.props;
-    const { name, iconUrl } = suggestion;
-
+    const { iconUrl, name, type } = suggestion;
```

Comment:
looks like a blank line has been removed. I like to see an empty line before a `return` statement.